IM_calculation/IM/im_calculation.py

```diff
@@ -250,7 +270,9 @@ def compute_measures_multiprocess(
     :param simple_output:
     :return:
     """
-    str_comps_for_int, str_comps = convert_str_comp(comp)
+    components_to_calculate, components_to_store = constants.Components.get_comps_to_calc_and_store(
+        comp
+    )
 
     bbseries, station_names = get_bbseis(input_path, file_type, station_names)
```

Comment:
This doesn't work for txt based waveforms?

Reply:
get_bbseis? It should do, I haven't touched it. The get comps to calc and store should also have the same behaviour as before